core/lib/dal/migrations/20240911161714_evm-simulator.down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE protocol_versions DROP COLUMN IF NOT EXISTS evm_simulator_code_hash;
+ALTER TABLE l1_batches DROP COLUMN IF NOT EXISTS evm_simulator_code_hash;
+ALTER TABLE miniblocks DROP COLUMN IF NOT EXISTS evm_simulator_code_hash;
```

Comment:
Is this still necessary given that the "up" migration doesn't create this column?